analytics/tests/test_views.py

```diff
@@ -398,73 +400,73 @@ def test_change_plan_type(self) -> None:
         cordelia_email = self.example_email("cordelia")
         self.login(cordelia_email)
 
-        result = self.client_post("/activity/support", {"realm_id": "1", "plan_type": "2"})
+        result = self.client_post("/activity/support", {"realm_id": "2", "plan_type": "2"})
```

Comment:
I don't think we want to use `2` here -- it seems like this should be the same `get_realm("zulip").id` thing?

(As a sidenote, this whole test file could be better migrated if we did `self.example_user("iago")` and then accessed `iago.realm` below -- less queries and complexity.

Reply:
(This sidenote applies to several test classes, I think).

Reply:
Oh, i missed this somehow, fixed now. The other tests use the realm Leary though, which is not the realm of iago, so I don't think it can be done this way?

Reply:
So, this comment was perhaps meant to apply to the whole branch.  If we want to access the `lear` realm, we can do `learn_realm=get_realm("lear")` or whatever.  

But in the vast majority of these tests, we actually already have a `self.zulip_realm` (for the management commands test I just fixed before merging), or a `user_profile.realm`, or whatever, and it's a lot cleaner to access those, since one doesn't need to think about whether the user belongs to the "zulip" realm or whatever -- it's accessing the same user object.